### DISCUSSION TITLE
autoprefixer in CSS Module doesn't work?

### DISCUSSION THREAD

**Discussion by lxthyme** [vercel/next.js#44585] [Help]
Hello:

   anyone can tell me what config can make autoprefixer worked in module css?

nextjs: 13.1.1

**Comment by icyJoseph** [comment]
I wonder if the problem is that since Next 13 the browserlist has been bumped:

> The supported browsers have been changed to drop Internet Explorer and target modern browsers. You can still use Browserslist to change targeted browsers.
> Chrome 64+
> Edge 79+
> Firefox 67+
> Opera 51+
> Safari 12+

Could you try to change it in your package json? https://nextjs.org/docs/basic-features/supported-browsers-features#browserslist, see if lowering the versions, or if adding back IE triggers the Postcss prefixes again.

**Reply by lxthyme** [reply]
it doesn't work. the same css style in js file autoprefixer worked. when i move them in css module file, it doesn't work.

**Comment by lxthyme** [comment]
module css:

![image](https://user-images.githubusercontent.com/8361463/210742312-ad174f02-eecc-405e-bd88-2a000d211cbb.png)

style jsx:

![image](https://user-images.githubusercontent.com/8361463/210742903-f5512e4e-e58f-42c9-a31b-53f5034e001f.png)
